draft-irtf-nmrg-network-digital-twin-arch.md

```diff
@@ -1292,30 +1300,82 @@ the implementation and deployment of digital twin network.
 
 # Security Considerations
 
-This document describes concepts and definitions of digital twin
-network. As such, the following security considerations are abstract
-and lack specific guidance or detail, i.e., in the form of principles,
-guidelines or requirements.
+This document describes concepts and definitions of digital twin networks.
+As this document presents system architecture, the following security
+considerations are abstract and lack specific guidance or detail, i.e.,
+principles, guidelines or requirements. However, the implementation and
+deployment of network digital twins will need to be carefully investigate
+the following security considerations, which may be categorized into
+different aspects:
 
-Security considerations of the Network Digital Twin include:
+* Data Management
 
-*  Secure the digital twin system itself.
+Synchronization: Synchronizing the data between the real and twin
+networks may increase the risk of sensitive data and information leakage.
 
-*  Data privacy protection.
+Data Access: Strict control and security mechanisms must be provided and
+enabled to prevent data leaks.
```

Comment:
```suggestion
enabled to prevent data leaks. Also, appropriate access rights must be provisioned to prevent unauthorized entities to access sensitive data (e.g., logging data used for legal data retention).
```